### DISCUSSION TITLE
Export plots to .avz format suitable for ANSYS Viewer.

### DISCUSSION THREAD

**Discussion by giannoko** [ansys/pyaedt#5385] [General]
Hi, 

Is it possible to directly export a plot to the .avz format, which is compatible with ANSYS Viewer? If not, are there any alternative methods or Python libraries that can help convert the plot to .avz format, either natively or through an intermediary conversion process?

Thank you

**Comment by Samuelopez-ansys** [comment]
Hi @anskgiannok, The only PyAnsys module that can export AVZ is this module:

https://github.com/ansys/pydynamicreporting

If I am not wrong you need to export the .case files.

**Reply by giannoko** [reply]
@Samuelopez-ansys  thank you for answer
